gulp/util/webpack-config.js

```diff
@@ -56,7 +56,6 @@ const EXTERNALS = {
     "jquery": "$",
     "moment": "moment",
     "react": "React",
-    "react-addons-css-transition-group": "React.addons.CSSTransitionGroup",
```

Comment:
Reverting seemed to do the trick 😃

Lemme know if this still looks weird and will take a more in-depth look.